creatingGame.js

```diff
@@ -16,46 +16,59 @@ function matriz() {
   jugador2.appendChild(textojugador2);
   players.appendChild(jugador2);
 
-  let numFilas = document.getElementById("filas").value;
-  let numColumnas = document.getElementById("columnas").value;
-
-  for (a = 0; a < numFilas; a++) {
-    let divPadre = document.querySelector(".padre");
-    let rowElement = document.createElement("div");
-    let atributo = document.createAttribute("class");
-    atributo.value = "row";
-    rowElement.setAttributeNode(atributo);
-    divPadre.appendChild(rowElement);
-
-    for (b = 0; b < numColumnas; b++) {
-      let columnElement = document.createElement("div");
-      let atributo2 = document.createAttribute("class");
-      atributo2.value = "column row" + [a + 1] + "-" + "column" + [b + 1];
-      columnElement.setAttributeNode(atributo2);
-      let imagen = document.createElement("img");
-      columnElement.appendChild(imagen);
-      rowElement.appendChild(columnElement);
-    }
-  }
+  let numRows = document.getElementById("filas").value;
+  let numColumns = document.getElementById("columnas").value;
+
+  let game = drawGame(numRows, numColumns);
+  let scores = game.scoresMatriz;
+  let positionsCunter = game.positionsCunter;
+
+  let winnerMessage = "";
+
   let padre = document.querySelector(".padre");
   padre.addEventListener("click", insertarX);
 
   var turno = jugador1;
+  let playCounter = 0;
 
   function insertarX(evento) {
+    playCounter = playCounter + 1;
     let imagen = evento.target;
+    let atributosImagen = imagen.getAttribute("class");
```

Comment:
this variable, should be const, and should be translate to english: `imageAttributes`